### PR TITLE
move vocabulary import to header.php

### PR DESCRIPTION
**Pull request by MuluhGodson**
## Fixes

Fixes https://github.com/creativecommons/project_creativecommons.org/issues/184 by @brylie 

## Description
This PR removes the vocabulary import from the root `style.css` to `header.php` so it's loaded only for the front end.

## Screenshots
Admin panel back to normal (unaffected by vocabulary)
image

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -13,7 +13,6 @@ This theme, like WordPress, is licensed under the GPL.
 
 @import url(https://unpkg.com/browse/bulma@0.9.3/);
 @import url(https://unpkg.com/@creativecommons/fonts@2020.9.3/css/fonts.css);
```
Comment (brylie) [inline style.css:15]: While we're here, should we go ahead and move these imports to the `header.php` as well? I.e., they're also not needed in the WordPress admin area.

Comment (MuluhGodson) [reply style.css:15]: I have this warning when the browser tries to load Bulma: 
```
Resource interpreted as Stylesheet but transferred with MIME type text/html: "https://unpkg.com/browse/bulma@0.9.3/".
```

Comment (MuluhGodson) [reply style.css:15]: can I use this link instead `https://cdn.jsdelivr.net/npm/bulma@0.9.3/css/bulma.min.css` as shown on the Bulma website

**Review by brylie — APPROVED** [review]

**Comment by MuluhGodson** [comment]
@brylie something to note:
We recently merged this PR:
- [x] https://github.com/creativecommons/vocabulary/pull/1023 which was the first task to resolve this issue:
- [x] https://github.com/creativecommons/project_creativecommons.org/issues/172

This PR imported `Bulma` as a CDN and I just realised that `minireset.css` is bundled by default with Bulma. This may be problematic as we may still have the same issues we had with `minireset.css` in Vocabulary. I opened the CDN version of Bulma and did a quick search of `minireset.css`:

image

**Comment by brylie** [comment]
One alternative is to define local styles that fix the ones reset by minireset.css.

**Pull request merged**